SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/LayeredLBStrategy.cpp

```diff
@@ -45,14 +45,23 @@ FString ULayeredLBStrategy::ToString() const
 	return Description;
 }
 
-void ULayeredLBStrategy::SetLayers(const TArray<FLayerInfo>& WorkerLayers)
+void ULayeredLBStrategy::SetLayers(TArray<FLayerInfo>& WorkerLayers)
 {
 	check(WorkerLayers.Num() != 0);
 
 	// For each Layer, add a LB Strategy for that layer.
 	for (int32 LayerIndex = 0; LayerIndex < WorkerLayers.Num(); ++LayerIndex)
 	{
-		const FLayerInfo& LayerInfo = WorkerLayers[LayerIndex];
+		FLayerInfo& LayerInfo = WorkerLayers[LayerIndex];
```

Comment:
Context for this - if the LBS for a layer becomes corrupted somehow (which has happened within Intrepid, possibly an artifact of P4 or engine upgrades), then without this change to set the LBS to the default, it crashes while launching the map. This is also the reason why the consts have been removed in the startup flow for the multi layer settings.